lang/main.json

```diff
@@ -466,6 +470,7 @@
         "loadingPeople": "Searching for people to invite",
         "noResults": "No matching search results",
         "noValidNumbers": "Please enter a phone number",
+        "notAvailable": "You can't invite people.",
```

Comment:
The strings for "toolbar" have been alphabetized per request. The new strings are "notAvailable", "shortcuts", "moreActions", and "feedback" (I might have missed some...). The "sharescreen" tooltip has been truncated because atlaskit tooltips get cutoff on the left edge of the screen, and the screenshare button is now on the far left of the horizontal toolbar. After discussing with Emil, we agreed this is an acceptable workaround.